.circleci/config.yml

```diff
@@ -33,32 +33,41 @@ jobs:
           name: Run Tests
           command: |
             . venv/bin/activate
+            mypy sprints
             coverage run -m pytest
             coverage report
+
+  deploy:
+    docker:
+      - image: circleci/python:3.7
+    steps:
       - setup_remote_docker:
           docker_layer_caching: true
       - run:
           name: Install Docker Compose
-          filters:
-            branches:
-              only: master
           command: |
             curl -L https://github.com/docker/compose/releases/download/1.19.0/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
             chmod +x ~/docker-compose
             sudo mv ~/docker-compose /usr/local/bin/docker-compose
       - run:
           name: Build and push Docker image
-          filters:
-            branches:
-              only: master
           command: |
             echo $DOCKER_PASSWORD | docker login -u $DOCKER_LOGIN --password-stdin
```

Comment:
@Agrendalath, do not forget to store these environment variables, particularly the ones which are secrets, in Vault, in case you havn't done it already.

Reply:
Ah, good point, I'll put them there :+1: